ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/StateTransition.java

```diff
@@ -226,15 +224,14 @@ public void process_slots(MutableBeaconState state, UnsignedLong slot, boolean p
             .plus(UnsignedLong.ONE)
             .mod(UnsignedLong.valueOf(SLOTS_PER_EPOCH))
             .equals(UnsignedLong.ZERO)) {
-          STATUS_LOG.log(
-              Level.INFO, "******* Epoch Event *******", printEnabled, StatusLogger.Color.BLUE);
+          STATUS_LOG.epochEvent();
           process_epoch(state);
           reportExceptions(CompletableFuture.runAsync(() -> recordMetrics(state)));
         }
         state.setSlot(state.getSlot().plus(UnsignedLong.ONE));
       }
     } catch (IllegalArgumentException e) {
-      STATUS_LOG.log(Level.WARN, e.getMessage());
+      LOG.warn(e.getMessage());
```

Comment:
We should include full stack traces whenever we move to a normal log4j logger.  Just make sure to include a string as the first arg and not just use `LOG.warn(e)` which will only print the message.